server/db/data-generator.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs');
+const faker = require('faker');
+const writer = require('csv-write-stream');
+const csv = writer();
+
+const generated = 10000000;
+let movieIdCounter = 1;
+let castIdCounter = -1;
+
+const movieCSV = fs.createWriteStream(
+  '/Users/dfuent/Desktop/SDC MODULE/castphotos/movies.csv',
+  {
+    encoding: 'utf8'
+  }
+);
+
+const generator = () => {
+  (async () => {
+    movieCSV.write('movie_id,cast_id, actor, character, image_url, role\n');
+    for (var i = 0; i < generated; i++) {
+      if (i % 24 === 0) {
+        movieIdCounter++;
```

Comment:
Avoid ++ operators to adhere to airbnb styleguide

Reply:
@mmcphillips refactored